README.md

```diff
@@ -1 +1,24 @@
-# couch-tracker-documentation
\ No newline at end of file
+# Couch-Tracker
+
+An Android application, to keep track of the TV shows you follow, and the movies you've watched.
+
+This repository contains all the documents and design decisions we've made.
+
+
+## Tenets
+
+- Free and open-source
+- No ads, trackers, subscriptions, etc.
+- Maintaining and developing the app can be an hobby: low (or zero) fees, low architectural complexity, supported by the community
```

Comment:
> can be an hobby

I don't know how much sense it makes to make this statement here. Agree on the "low (or zero) fees, low architectural complexity", but the hobby part should be just for us. Tenents of the app is something user should be able to read and understand, this is out of scope for a public doc. Also, anything free and open-source by design is not set up to be supported like a paid service, this doesn't need to be said.

Reply:
Updated

Reply:
I'd say binary releases could come with default API keys. Building the app will require a key (either at build time, or the user will need to enter it explicitly in the app's settings).

Out of scope of this section though. We can add a specific section about external data.